### ISSUE TITLE
The source list for Content Security Policy directive 'script-src' contains an invalid source: ''strict-dynamic''. It will be ignored.

### ISSUE THREAD

**Issue by lukepighetti** [AurityLab/recaptcha-v3#112]
On safari we're seeing this error

```
The source list for Content Security Policy directive 'script-src' contains an invalid source: ''strict-dynamic''. It will be ignored.
```

It appears to be connected to `vue-recaptcha-v3` but unfortunately I do not have a MVCE available for reproduction. 

Any thoughts?

**Comment by WipeAir** [comment]
Hi,
I've moved your issue to the reCAPTCHA-v3 repository, as vue-reCAPTCHA-v3 just wraps it.

Looks like this [issue](https://github.com/google/google-api-javascript-client/issues/397) also occurs on the official client.
Does it prevent loading the reCAPTCHA or using it?

**Comment by mesqueeb** [comment]
@thefoxie were you able to find any solution? I have the same issue yesterday in a vue + capacitor app in the iOS webview.

**Comment by thefoxie** [comment]
@mesqueeb yeah I got it working as I remember. Not sure how though. If I remember correctly my app didn't work because of some strange <select> behaviour in webkit (used epiphany aka gnome web for debugging) but all I could see was just this error in console.

Here is how I used the vue-recaptcha-v3 plugin:

https://github.com/thefoxie/LepsieSluzby.sk/blob/master/src/index.js#L13
https://github.com/thefoxie/LepsieSluzby.sk/blob/master/src/views/podnet.vue#L236
